corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1008,6 +1029,12 @@ def column_fieldset(self):
     @memoized
     def initial_columns(self):
         if self.existing_report:
+            reverse_agg_map = {
+                'avg': 'Average',
+                'sum': 'Sum',
+                'count': 'Count',
+                'simple': 'No Aggregation'
+            }
             cols = []
```

Comment:
@calellowitz 

Can you update the aggregation types and names? I think we want to support:
'avg': 'Average',
'sum': 'Sum'
'expanded': 'Count Per Choice'

I think Simple aggregation is implied by the type of report, or the group by option. 

https://docs.google.com/document/d/1Xh9QIlVyBT2ujLoCxkVybG__5JhbJ5Y48cNj3SfRmz8/edit#